src/core/lib/iomgr/executor.h

```diff
@@ -34,20 +34,17 @@ typedef struct {
   bool shutdown;
   bool queued_long_job;
   grpc_core::Thread thd;
-} thread_state;
+} ThreadState;
 
-typedef enum {
-  GRPC_EXECUTOR_SHORT,
-  GRPC_EXECUTOR_LONG
-} grpc_executor_job_length;
+typedef enum { GRPC_EXECUTOR_SHORT, GRPC_EXECUTOR_LONG } grpc_executor_job_type;
```

Comment:
nit: same as with `ThreadState`, we may want to rename `grpc_executor_job_type` to `GrpcExecutorJobType`. However, given that it's a "public" name, we may not want to do that right now.